Presentation.Web/Tests/Models/systemDeleteMessages.ts

```diff
@@ -0,0 +1,10 @@
+﻿class SystemDeleteMessages {
```

Comment:
Det er stadig bare en error message container. Hvorfor ikke dedikere denne klasse til errorMessages, så man kan skrive SystemDeleteErrorMessages.systemInUse osv?

Reply:
Har du brug for at denne er en string container i test context??? Du vil jo typisk kun bruge den i den test hvor du tester teksterne. Hvorfor bor de ikke bare i dén "expect"?